src/test/rpc/Subscribe_test.cpp

```diff
@@ -872,7 +872,7 @@ class Subscribe_test : public beast::unit_test::suite
             for (int i = 0; i < numReplies; ++i)
             {
                 std::uint32_t idx{0};
-                auto reply = wsc.getMsg(100ms);
+                auto reply = wsc.getMsg(200ms);
```

Comment:
This is an unrelated fix and should be separate (and probably to `develop` if this is to fix that flaky test)